src/transformStateWithClones.js

```diff
@@ -8,6 +8,36 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const stateCopy = { ...state };
+  const arrayOfStates = [];
```

Comment:
тип змінної в назві змінної вказувати не оч мені кажеться ))
```suggestion
  const states = [];
```